[2024-08-08T15:34:04.948+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-08-08T15:34:04.966+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_prueba_tecnica.limpieza manual__2024-08-08T15:33:45.003080+00:00 [queued]>
[2024-08-08T15:34:04.970+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_prueba_tecnica.limpieza manual__2024-08-08T15:33:45.003080+00:00 [queued]>
[2024-08-08T15:34:04.971+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 6
[2024-08-08T15:34:04.980+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): limpieza> on 2024-08-08 15:33:45.003080+00:00
[2024-08-08T15:34:04.987+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62: DeprecationWarning: This process (pid=361) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-08-08T15:34:04.987+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag_prueba_tecnica', 'limpieza', 'manual__2024-08-08T15:33:45.003080+00:00', '--job-id', '42', '--raw', '--subdir', 'DAGS_FOLDER/dag_prueba_tecnica.py', '--cfg-path', '/tmp/tmpj6705fk4']
[2024-08-08T15:34:04.988+0000] {standard_task_runner.py:64} INFO - Started process 374 to run task
[2024-08-08T15:34:04.988+0000] {standard_task_runner.py:91} INFO - Job 42: Subtask limpieza
[2024-08-08T15:34:05.019+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_prueba_tecnica.limpieza manual__2024-08-08T15:33:45.003080+00:00 [running]> on host a8ee84aadfbc
[2024-08-08T15:34:05.078+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='miguel_guimarey' AIRFLOW_CTX_DAG_ID='dag_prueba_tecnica' AIRFLOW_CTX_TASK_ID='limpieza' AIRFLOW_CTX_EXECUTION_DATE='2024-08-08T15:33:45.003080+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-08-08T15:33:45.003080+00:00'
[2024-08-08T15:34:05.079+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-08-08T15:34:08.068+0000] {warnings.py:112} WARNING - /opt/***/dags/dag_prueba_tecnica.py:40: FutureWarning: Passing literal json to 'read_json' is deprecated and will be removed in a future version. To read from a literal string, wrap it in a 'StringIO' object.
  df = pd.read_json(data, orient='records')

[2024-08-08T15:34:18.274+0000] {base.py:84} INFO - Using connection ID 'postgres_localhost' for task execution.
[2024-08-08T15:34:52.901+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-08-08T15:34:52.902+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-08-08T15:34:52.909+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dag_prueba_tecnica, task_id=limpieza, run_id=manual__2024-08-08T15:33:45.003080+00:00, execution_date=20240808T153345, start_date=20240808T153404, end_date=20240808T153452
[2024-08-08T15:34:53.071+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2024-08-08T15:34:53.083+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-08-08T15:34:53.086+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
